src/stcal/jump/jump.py

```diff
@@ -514,6 +532,21 @@ def detect_jumps(
     # Return the updated data quality arrays
     return gdq, pdq, total_primary_crs, number_extended_events, stddev
 
+# Find micrometeorite flashes that light up the entire array in a small number
+# of groups.  Do this by looking for cases where greater than mmflash_pct percent
+# of the array is flagged as 'JUMP_DET' and flag the entire array.
+def find_micrometeorite_flashes(gdq, jump_flag, mmflashfrac):
```

Comment:
This is missing a docstring.

Also, start the function with `nints, ngroups, nrows, ncols = gdq.shape`, then use these more descriptive variable names, rather than `shape[k]`.

Reply:
Good point- done